src/plugins/discover/public/application/components/data_grid/data_grid_table_cell_value.test.tsx

```diff
@@ -158,6 +158,7 @@ describe('Testing fetchTableDataCell function', () => {
       >
         <EuiDescriptionListTitle
           className="osdDescriptionListFieldTitle"
+          data-test-subj="osdDescriptionListFieldTitle"
```

Comment:
I understand that the names were made to match the `className` values but sometimes `className` values are shared across components. However, it would be best if test subject names were unique and prefixed to avoid collisions.

Maybe something like:
```suggestion
          data-test-subj="dscDataGridTableCellListFieldTitle"
```